common/protob/messages-cardano.proto

```diff
@@ -50,22 +51,21 @@ message CardanoPublicKey {
  * Request: Ask device to sign Cardano transaction
  * @start
  * @next CardanoSignedTx
- * @next CardanoTxRequest
  * @next Failure
  */
 message CardanoSignTx {
     repeated CardanoTxInputType inputs = 1;     // inputs to be used in transaction
     repeated CardanoTxOutputType outputs = 2;   // outputs to be used in transaction
-    optional uint32 transactions_count = 3;     // transactions count
-    optional uint32 protocol_magic = 5;         // network's protocol magic
+    optional uint32 protocol_magic = 3;         // network's protocol magic
```

Comment:
IIRC reusing protobuf field ids is a no-no because if somebody sends a message following the old convention it may clash.

Was there a reason why you changed the id of the "protocol_magic" field?

Reply:
http://itshark.xyz/posts/2018/7/11/Protocol_Buffers_basic_stuff_you_need_to_know -> `It is very important to remember that once some number is used in some message, it can never be changed or reused.`

Reply:
I would've thought that it is only a problem if you want to keep some support for the old version. Since we're pretty much dropping support of the old version I thought it'd be OK. But sure, better safe than sorry - I'll fix it.